converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Microsoft.Health.DicomCast.Core.csproj

```diff
@@ -22,7 +22,7 @@
     <PackageReference Include="Microsoft.Health.Client" Version="$(HealthcareSharedPackageVersion)" />
     <PackageReference Include="Microsoft.Health.Core" Version="$(HealthcareSharedPackageVersion)" />
     <PackageReference Include="Microsoft.Health.Extensions.DependencyInjection" Version="$(HealthcareSharedPackageVersion)" />
-    <PackageReference Include="Microsoft.Health.Fhir.R4.Client" Version="1.0.0-master-20201110-7" />
+    <PackageReference Include="Microsoft.Health.Fhir.R4.Client" Version="1.0.869" />
```

Comment:
I had to upgrade this package because it used to (mistakenly) include StyleCop as a dependency, which forced us to StyleCop in this project lol!